ztp/resource-generator/README.md

```diff
@@ -0,0 +1,88 @@
+# README #
+
+This directory provides RAN site ArgoCD gitops pipeline container image definitions
+
+## What is this repository for? ##
+### Quick summary ###
+
+The idea is to accept RAN site manifests in a format defined by "siteconfig" CRD, convert them to custom resources accepted by the assisted service and ACM, and apply them to the cluster running Assisted installer operator and ACM hub.
```

Comment:
Replace "siteconfig" with SiteConfig
Assisted installer is the Assisted Service Operator